RELEASE/scripts/sl_ascend/sl_cooking.ash

```diff
@@ -1768,6 +1773,11 @@ boolean loadConsumables(string _type, ConsumeAction[int] actions)
 			{
 				pullables[it] = min(howmany, min(pulls_remaining(), storage_amount(it)));
 			}
+			boolean[item] KEY_LIME_PIES = $items[Boris's key lime pie, Jarlsberg's key lime pie, Sneaky Pete's key lime pie];
```

Comment:
Would they not be handled under `SL_OBTAIN_BUY` already?

Reply:
Nope, SL_OBTAIN_BUY is buying from coinmasters/shops. SL_OBTAIN_PULL handles the "maybe we need to mallbuy this item" case when it calls `pullXWhenHaveY()`.